core/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -387,7 +387,7 @@ export class TransformerExtensiveTestScenario {
     assert.isTrue(displayStyle3d.settings.hasSubCategoryOverride);
     assert.equal(displayStyle3d.settings.subCategoryOverrides.size, 1);
     assert.exists(displayStyle3d.settings.getSubCategoryOverride(subCategoryId), "Expect subCategoryOverrides to have been remapped");
-    assert.isTrue(displayStyle3d.settings.excludedElements.has(physicalObjectId1), "Expect excludedElements to be remapped"); // eslint-disable-line deprecation/deprecation
+    assert.isTrue(new Set<string>(displayStyle3d.settings.excludedElementIds).has(physicalObjectId1), "Expect excludedElements to be remapped"); // eslint-disable-line deprecation/deprecation

```

Comment:
shouldn't you be able to remove the eslint-ignore comment now that you're no longer using anything deprecated?
also, do we not have a way to fail on unnecessary eslint ignores?

Reply:
yes, overlooked. no, we don't have a lint rule to complain about unnecessarily disabling lint rules.

Reply:
fixed.

Reply:
this line doesn't appear to have received the fix, no?

Reply:
try now.